contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -89,6 +88,26 @@ void multisig::approve( name proposer, name proposal_name, permission_level leve
             a.requested_approvals.erase( itr );
          });
    }
+
+   transaction_header trx_header = get_trx_header(prop.packed_transaction.data(), prop.packed_transaction.size());
+
+   if ( prop.earliest_exec_time.has_value() ) {
+      if ( prop.earliest_exec_time.value().has_value() ) {
+         return;
+      } else {
```

Comment:
Rather than the `return` in the line above, I would like to see this else branch instead guarded by `if( !prop.earliest_exec_time->has_value() ) {`.

Reply:
Previously linked: https://github.com/EOSIO/eosio.contracts/blob/deprecate-deferred3/contracts/eosio.msig/src/eosio.msig.cpp#L107-L116